shared/src/main/scala/uk/gov/homeoffice/drt/ShiftMeta.scala

```diff
@@ -0,0 +1,8 @@
+package uk.gov.homeoffice.drt
+
+case class ShiftMeta(
+                      port: String,
+                      terminal: String,
+                      shiftAssignmentsMigratedAt: Option[java.sql.Timestamp],
```

Comment:
We don't want sql.Timestamp outside of a table or serialisation

Reply:
you mean to have string rather than timestamp ? or just boolean saying it exists